.werft/jobs/build/prepare.ts

```diff
@@ -17,8 +18,23 @@ export async function prepare(werft: Werft) {
         exec("gcloud auth configure-docker --quiet");
         exec("gcloud auth configure-docker europe-docker.pkg.dev --quiet");
         exec('gcloud container clusters get-credentials core-dev --zone europe-west1-b --project gitpod-core-dev');
-        werft.done('prep');
+
+        if (lookForBranchNameWithVM(config.repository.branch)) {
+            config.withVM = true
+        }
     } catch (err) {
         werft.fail('prep', err);
     }
+
+    werft.done('prep');
+    return config
+}
+
+function lookForBranchNameWithVM(branchName: string): boolean {
+    return branchName.includes("with-vm") ||
```

Comment:
Just just stick with `with-vm`. Currently the load-test is the only consumer of this so lets keep it simple ☺️